CMakeLists.txt

```diff
@@ -388,6 +388,45 @@ set ( math_headers
       src/math/vector_4d.hpp
     )
 
+
+set ( scripting_sources
+    src/noggit/scripting/script_random.cpp
```

Comment:
I usually do this specifically for scripting APIs because it makes them easier to spot and search for as they tend to overlap with files from other packages, but that's really more a problem with my editor than anything else. I can change them if you still want to.